src/math.rs

```diff
@@ -276,19 +301,43 @@ impl<S: BaseFloat> Obb<S> {
     }
 }
 
+impl<S> PointCulling<S> for Obb<S>
```

Comment:
Nit: Why was this moved?

Reply:
the methods have been moved to implement the trait - I hope to understand the question